core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/Driver.java

```diff
@@ -326,8 +320,8 @@ private static String getBarcode(Barcodable barcodable) {
     return str;
   }
 
-  private static String getBarcode64(Barcodable barcodable) throws UnsupportedEncodingException {
-    return new String(Base64.encodeBase64(getBarcode(barcodable).getBytes("UTF-8")));
+  private static String getBarcode64(Barcodable barcodable) {
+    return new String(Base64.encodeBase64(getBarcode(barcodable).getBytes(StandardCharsets.UTF_8)));
   }
 
   private static int multiline(int lineLength, int maxLines, String input, BiConsumer<Integer, String> writer) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this method into the anonymous class declared at line 69. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3398)